### PR TITLE
chore: do not show sensitive certificate details in Terraform cmd out…

### PR DESCRIPTION
**Pull request by feng-zhe**
Mark some certificate-related information as sensitive so that they wouldn't be shown in the Terraform cmd output like `terraform destroy`.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
sql: update `replica_configuration`, `ca_cert`, and `server_ca_cert` fields to be flagged sensitive in `google_sql_instance` and `google_sql_ssl_cert` resources
```

### CODE REVIEW

File: mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.erb [hunk 1]
```diff
@@ -775,11 +775,12 @@ is set to true. Defaults to ZONAL.`,
...
-				Type:     schema.TypeList,
-				Optional: true,
-				MaxItems: 1,
+				Type:       schema.TypeList,
+				Optional:   true,
+				MaxItems:   1,
 				// Returned from API on all replicas
-				Computed: true,
+				Computed:   true,
+				Sensitive:  true,
```
Comment (NickElliot) [inline mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.erb:783]: The individual sub fields that specifically contain the encrypted data should be flagged sensitive for this update, not the entire list object

File: mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.erb [hunk 2]
```diff
@@ -865,8 +866,9 @@ is set to true. Defaults to ZONAL.`,
 				Description: `The configuration for replication.`,
 			},
 			"server_ca_cert": {
-				Type:     schema.TypeList,
-				Computed: true,
+				Type:       schema.TypeList,
+				Computed:   true,
+				Sensitive:  true,
```
Comment (NickElliot) [inline mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.erb:871]: ditto here

**Review by NickElliot — CHANGES_REQUESTED** [review]
marked a couple needed changes for this implementation

File: mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.erb (hunk 2 above)
Comment (feng-zhe) [reply mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.erb:871]: Thanks, Nick. Unfortunately this is a limitation of TF for these nested messages. Please see [this issue](https://github.com/hashicorp/terraform-plugin-sdk/issues/201). So I have to mark the whole message as sensitive.

I've tested the version which only marks the sensitive fields in the nested message. TF failed to hide them.

File: mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.erb (hunk 1 above)
Comment (feng-zhe) [reply mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.erb:783]: Thanks, Nick. Unfortunately this is a limitation of TF for these nested messages. Please see https://github.com/hashicorp/terraform-plugin-sdk/issues/201. So I have to mark the whole message as sensitive.

I've tested the version which only marks the sensitive fields in the nested message. TF failed to hide them

**Comment by feng-zhe** [comment]
Friendly ping. Thanks.

**Review by NickElliot — APPROVED** [review]
LGTM then!

**Pull request merged**